lib/content-services/src/lib/viewer/components/alfresco-viewer.component.ts

```diff
@@ -215,6 +215,10 @@ export class AlfrescoViewerComponent implements OnChanges, OnInit, OnDestroy {
     nodeEntry: NodeEntry;
     tracks: Track[] = [];
     readOnly: boolean = true;
+    allowedActions: { [key: string]: boolean } = {
```

Comment:
I've implemented the new `allowedActions` property which lets us specify which actions should be available when not in read-only mode.

For `hxviewer`, we can configure it like this:

```
<adf-viewer
    ...
    [readOnly]="false"
    [allowedActions]="{ rotate: true, crop: false }"
    ...>
</adf-viewer>
```

This setup ensures that in `hxviewer`, users can rotate but not crop, even when the viewer is not in read-only mode.